k8s/examples/wordpress/wordpress/20-statefulset.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
This should be a Deployment. If the container image is build properly, only the wp-content/uploads require to be mounted with the shared volume.

The whole point of having SharedFS is to be able to scale horizontally a deployment. 

It is also recommended to serve the wp config file either baked in the image or as a configmap (wp-config.php). So the salts to sign the cookies when authentication happens match between instances of the application.  
I can clarify on this point if needed  =)